pages/Campaign/index.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { useEffect, useRef, useState } from "react";
+import Image from "next/image";
+import { Toaster, toast } from "sonner";
+import {
+  CampaignFormData,
+  validateField,
+  ValidateForm,
+} from "../../Schemas/CampaignSchema";
+import axios from "axios";
+import { ApiUrls } from "@/Configurations/ApiUrls";
+import {
+  CampaignCustomizationModel,
+  CampaignCustomizationsModel,
+  defaultCampaignCustomizations,
+} from "@/Models/CampaignModel";
+import { MetaProps } from "@/Models/MetaProps";
+import Meta from "../../components/Meta";
+import CampaignSkelton from "@/components/CampaignSkelton";
+const defaultFormData: CampaignFormData = {
+  Name: "",
+  CompanyName: "",
+  ContactNumber: "",
+  CompanyEmailId: "",
+  Comments: "",
+};
+const metaData: MetaProps = {
+  title: "Campaign Landing Page",
+  description: "To Submit the Campaign Form Data",
+  keywords: "medmonk, campaign",
+};
+const Campaign: React.FC = () => {
+  const timerRef = useRef<any>(null);
+  const [formData, setFormData] = useState<CampaignFormData>(defaultFormData);
+  const [errors, setErrors] = useState<Partial<CampaignFormData>>({});
+  const [customizations, SetCustomizations] =
+    useState<CampaignCustomizationModel>(defaultCampaignCustomizations);
+  const [submitting, setSubmitting] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  useEffect(() => {
+    const headers = {
+      "Content-Type": "application/json",
+    };
+    axios
+      .get<CampaignCustomizationsModel>(
+        `${ApiUrls.CampaignCustomizations}?populate=*`,
+        { headers }
+      )
+      .then((response) => {
+        setIsLoading(false);
+        if (
+          response.data &&
+          response.data &&
+          response.data.data &&
+          response.data.data.length > 0
+        ) {
+          let customizationAttribute = response.data.data[0].attributes;
+          if (customizationAttribute) {
+            if (customizationAttribute.Banner.data.attributes.url) {
+              customizationAttribute.Banner.data.attributes.url =
+                ApiUrls.BaseAddress +
+                customizationAttribute.Banner.data.attributes.url;
+            }
+            SetCustomizations(customizationAttribute);
+          }
+        }
+      })
```

Comment:
do not use react-hooks, as its a next-js use nextjs methods such as getStatic props, getstatic etc. CHeck NextJs documetation properly and follow the standards propelry please. Lets not use React code within NextJs